tests/test_statistics.py

```diff
@@ -114,3 +114,16 @@ def test_bincount_weights(self, array, weights):
         expected = numpy.bincount(np_a, weights=weights)
         result = dpnp.bincount(dpnp_a, weights=weights)
         numpy.testing.assert_array_equal(expected, result)
+
+def test_cov_rowvar1():
+    a = dpnp.array([[0, 2], [1, 1], [2, 0]],device='cpu')
```

Comment:
```suggestion
    a = dpnp.array([[0, 2], [1, 1], [2, 0]])
```

I think you do not need to set `device='cpu'` here. Just choose a default selected device.